### ISSUE TITLE
feat (gui) : pdf preview in high resolution

### ISSUE THREAD

**Issue by timelic** [Byaidu/PDFMathTranslate#290]
改进方法很简单：

```py
# gui.py
preview = PDF(label="Document Preview", visible=True, height=2000) # 设置一个比较大的数值
```

然后添加一个 CSS：

```css
.pdf-canvas canvas {
    width: 100%;
}
```

<img width="982" alt="截屏2024-12-19 11 04 09" src="https://github.com/user-attachments/assets/ab025b58-4eaf-4572-91c6-620dc4bea184" />

**Comment by Byaidu** [comment]
感谢支持！想问下有兴趣提交pr吗，这样贡献者列表就可以多一位

**Comment by reycn** [comment]
The feature has been implemented in [PR#294](https://github.com/Byaidu/PDFMathTranslate/pull/294) by @hellofinch, with all CI tests passed.
